integration/build/build_test.go

```diff
@@ -524,6 +524,7 @@ RUN for g in $(seq 0 8); do dd if=/dev/urandom of=rnd bs=1K count=1 seek=$((1024
 }
 
 func TestBuildWCOWSandboxSize(t *testing.T) {
+	skip.If(t, testEnv.OSType == "windows", "FIXME. Hang with containerd")
```

Comment:
Was this actually a hang? It can be a slow (20 minutes, if I remember correctly) test, due to the time taken to commit each layer. That shouldn't be affected by containerd, since we don't use the containerd WCOW support for that (as it doesn't exist yet...). The `fsutil` calls themselves should be super-fast, they are literally creating a single file with a length but no data.

Reply:
Ok. Might be that case then. I think that TestExecWithCloseStdin was original issue which made CI failing. These two others I looked locally just couple of minutes and assumed that they hang.

Reply:
Since it's not doing anything particularly interesting for this PR, it's probably worth leaving disabled (but maybe with a clearer message and a 'do not merge') until other issues are sorted out, so you get quicker CI turnaround.

Reply:
```
[2021-03-18T11:25:52.710Z] === RUN   TestBuildWCOWSandboxSize
[2021-03-18T11:42:01.271Z] --- PASS: TestBuildWCOWSandboxSize (962.72s)
```

Only 16 minutes. Faster than I remember. ^_^